Exceptions/Exceptions/Program.cs

```diff
@@ -0,0 +1,82 @@
+﻿namespace Exceptions
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+
+            //Basic Exception handling
+            try
+            {
+                Console.WriteLine("Enter first number: ");
+                int num1 = GetUserInput();
+
+                Console.WriteLine("Enter second number: ");
+                int num2 = GetUserInput();
+
+                if (num2 == 0)
+                {
+                    throw new DivideByZeroException("Cannot divide by zero");
```

Comment:
@VikramSunke The DivideByZeroException is to throw when attempting to divide by zero. Instead of throwing an exception, it should display a valid message when "0" is given as the second input.

Reply:
I'll update that Yogesh